src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -49,6 +49,18 @@ public boolean isValidViewer() {
     public boolean isTeam() {
         return this == TEAMS || this == OWN_TEAM;
     }
+    
+    public boolean isOwnTeamMembers() {
+        return this == OWN_TEAM_MEMBERS;
```

Comment:
If SELF is included in the recipient than student will be displayed the question even if he have no Team members, we need not to take care of that. Like
![screenshot from 2016-07-19 13 09 16](https://cloud.githubusercontent.com/assets/12219700/16942143/1dea2b8a-4db2-11e6-9644-34c3913c9c72.png)